documentation/book/con-user-operator-cluster-label.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// assembly-getting-started-user-operator.adoc
+
+[id='con-user-operator-cluster-label-{context}']
+
+= Identifying a Kafka cluster for user handling
+
+A `KafkaUser` resource includes a label that defines the appropriate name of the Kafka cluster (derived from the name of the `Kafka` resource) to which it belongs.
+
+[source,yaml,subs="attributes+"]
+----
```

Comment:
I suggest adding the start of the YAML file for context, e.g.

```
apiVersion: kafka.strimzi.io/v1beta1
kind: KafkaUser
metadata:
  name: my-user
```

Reply:
Too minimal? :smiley: Added